test/realm_freeipa/integration_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+require 'json'
+require 'root/root_v2_api'
+# require 'realm_freeipa/configuration_loader'
```

Comment:
Could you remove the commented code here?

Reply:
oops, fixed